pkg/meta/model/job_args.go

```diff
@@ -195,6 +196,50 @@ type TruncateTableArgs struct {
 	OldPartIDsWithPolicy []int64 `json:"-"`
 }
 
+// RenameTablesArgs is the arguments for rename tables job.
+type RenameTablesArgs struct {
+	OldSchemaIDs   []int64         `json:"old_schema_ids,omitempty"`
```

Comment:
for those slices, they should have same length, better use a slice of `RenameTableArgs` inside it, see https://github.com/pingcap/tidb/pull/55959/files#diff-0a328bfc50df4984d8b701f6cfb588c8bc675992e32e00ae05f2d900147ea8dfR115

Reply:
But we still need to store new `OldTableNames` and `TableIDs` in slice if using `RenameTableArgs`? Maybe I could add new struct for each table.

Reply:
> new OldTableNames and TableIDs in slice

why is that? for each table to rename, there should be only 1 OldTableName and ID

Reply:
> for each table to rename, there should be only 1 OldTableName and ID

But these infos are not stored in `RenameTableArgs`?

Reply:
we can add OldTableName and TableID to RenameTableArgs, quite small